pkg/networkmanager/interfaces.js

```diff
@@ -1612,7 +1612,7 @@ PageNetworking.prototype = {
         PageNetworkBondSettings.settings =
             {
                 connection: {
-                    id: uuid,
+                    id: iface,
```

Comment:
The id needs to be updated when people change the interface name in the dialog, here and elsewhere.